installer.sh

```diff
@@ -1,10 +1,57 @@
 #!/usr/bin/env bash
+
+systemctl status ModemManager | grep 'active (running)' > /dev/null 2>&1
+
+if [ $? != 0 ]
+then
+        sudo systemctl disable ModemManager > /dev/null
+        sudo systemctl stop ModemManager > /dev/null
+fi
+
 set -e
 
 function info { echo -e "[Info] $*"; }
 function error { echo -e "[Error] $*"; exit 1; }
 function warn  { echo -e "[Warning] $*"; }
 
+# Install required dependencies
+test $? -eq 0 || exit 1 "you should have sudo priveledge to run this script"
```

Comment:
Why test `$?` ?

Probably better to use the `EUID` variable that is natively available, if that is not 0, exit using an `error()` function call to display a message and exit with a non-zero error.

Reply:
I'm not sure how to implement this change.